### PR TITLE
added github id to lists report

### PR DESCRIPTION
**Pull request by call**
Adding github id to the report.

### CODE REVIEW

File: lib/maintainers/runner.rb [hunk 1]
```diff
@@ -183,13 +183,12 @@ def report_lists_details(maintainers_files)
...
         maintainers_files.keys.sort.each { |repo|
           maintainers = JSON.load( maintainers_files[repo] )
-          list = maintainers['internal_list']
-          if list 
-            maintainers['people'].each { |person|
-              name =  "#{person['email'] ? person['email'] : person['name'] ? person['name'] : person['github']}"
-              csv << [repo,list, name ]
-            }
-          end
+          list = "#{maintainers['internal_list']}"
```
Comment (MikaelSmith) [inline lib/maintainers/runner.rb]: Why did this change?

File: lib/maintainers/runner.rb [hunk 2]
```diff
@@ -183,13 +183,12 @@ def report_lists_details(maintainers_files)
...
+          maintainers['people'].each { |person|
+            name =  "#{person['email'] ? person['email'] : person['name'] ? person['name'] : person['github']}"
+            github = "#{person['github']}"
+            csv << [repo, list, name, github]
```
Comment (MikaelSmith) [inline lib/maintainers/runner.rb:190]: @theshanx is this something you need to know about?

**Review by MikaelSmith — CHANGES_REQUESTED** [review]

File: lib/maintainers/runner.rb (hunk 2 above)
Comment (theshanx) [reply lib/maintainers/runner.rb:190]: This should make it easier for IT to manage github teams - my only concern is with  line 183 - I think @bcall needs a header title...

**Comment by call** [comment]
I've added the missing header label for the 'github' column. Does this look ok now?

**Review by MikaelSmith — APPROVED** [review]

**Pull request merged**